pslab/bus/i2c.py

```diff
@@ -48,6 +58,45 @@ def __init__(self, device: SerialHandler = None):
         self._running = False
         self._mode = None
 
+    def _init(self):
+        self._device.send_byte(CP.I2C_HEADER)
+        self._device.send_byte(CP.I2C_INIT)
+        self._device.get_ack()
```

Comment:
Is there any reason for this method to be called after object instantiation? If not, I would prefer this being in the actual `__init__` method.

Reply:
If we move `I2CPrimitive._init` to `I2CPrimitive.__init__` then I2C will be initialized every type I2CSlave object is created.